Events transcript (#3862)
DFE-Digital/get-into-teaching-app: app/views/content/events/what-happens-at-events-transcript.md

[Video shows teaching presentations and stalls in the 'training provider zone' at the event.]

**Speaker 1:**

It’d be really interesting to hear how you found today’s event.

**Attendee 1:** 

I found it interesting and informative. I didn’t realise there were that many options to go down.

**Attendee 2:**

I’ve definitely found speaking to the teachers most useful and just having some questions prepared and speaking to them directly, just to find out exactly what I need to know.

**Attendee 3:**

I came here looking for more advice, maybe, possibly some internships perhaps and anything that’s interesting that I should know of before getting into the world of teaching.

**Attendee 4:**

I am a student at University of Birmingham. We didn’t really know what provider or what route to go down. I’ve always been set on doing it through the university based [approach], but talking to the advisers, talking to the relevant teachers, they really did open my eyes to the possibilities of going through a school direct approach.

**Attendee 5:**

> In a couple years' time I’d like to be a teacher, and how I get to that point, and they pointed me the right directions, so I’ve managed to find and speak to the right people during the exhibition and find the right providers and I feel like super confident now to go forward and start applying.

Reviewer: ```suggestion
In a couple years' time I’d like to be a teacher, and how I get to that point, and they pointed me the right directions, so I’ve managed to go and speak to the right people during the exhibition and find the right providers and I feel like super confident now to go forward and start applying.
```

---
title: What happens at a Get Into Teaching event video transcript
description: Full transcript for the video 'What happens at a Get Into Teaching event' video.
---

**Speaker 1 - Georgie Butler:** 

Here we are in Manchester for the first Get Into Teaching event of the series.
If you’ve ever wondered what happens at a Get Into Teaching event, let’s go take a look.

> [Video shows people and stalls at the event, including the 'advice zone' and 'teacher hub']

Reviewer: ```suggestion
[Video shows people and stalls at the event, including the 'advice zone' and 'teacher hub'.]
```

**Speaker 1:** 

These Get Into Teaching events will provide information, advice and support to anyone looking into a career in teaching. 

**Speaker 2 - Amanda Bartley, On-campus Marketing Manager, Department for Education:** 

The purpose of this event is to get people like you into teaching, to inspire the next generation. Teachers are fundamental to our society, so we want to help you on your journey to becoming a teacher.

> [Video shows teaching presentations and stalls in the 'training provider zone' at the event]

Reviewer: ```suggestion
[Video shows teaching presentations and stalls in the 'training provider zone' at the event.]
```

**Speaker 1:**

It’d be really interesting to hear how you found today’s event.

> **Attendee 3:** 

Reviewer: Technically, wouldn't they be Attendee 1? In which case, all the attendee numbers would need decreasing by 2?

But if you think this is too confusing to have a Speaker 1 and an Attendee 1, let's leave as is!

I found it interesting and informative. I didn’t realize there were that many options to go down.

**Attendee 4:**

I’ve definitely found speaking to the teachers most useful and just having some questions prepared and speaking to them directly, just to find out exactly what I need to know.

**Attendee 5:**

I came here looking for more advice, maybe, possibly some internships perhaps and anything that’s interesting that I should know of before getting into the world of teaching.

**Attendee 6:**

I am a student at University of Birmingham. We didn’t really know what provider or what route to go down. I’ve always been set on doing it through the university based [approach], but talking to the advisers, talking to the relevant teachers, they really did open my eyes to the possibilities of going through a school direct approach.

**Attendee 7:**

> In a couple years time I’d like to be a teacher, and how I get to that point, and they pointed me the right directions, so I’ve managed to find and speak to the right people during the exhibition and find the right providers and I feel like super confident now to go forward and start applying.

Reviewer: ```suggestion
In a couple of years' time I’d like to be a teacher, and how I get to that point, and they pointed me the right directions, so I’ve managed to find and speak to the right people during the exhibition and find the right providers and I feel like super confident now to go forward and start applying.
```
